### ISSUE TITLE
"warning: explicit link request could not be resolved" - but I didn't explicitly request a link

### ISSUE THREAD

**Issue by eyalroz** [doxygen/doxygen#10713]
**Describe the bug**
I have the following doxygen comment for some method of mine:
```
/**
 * @brief Create a variant of ::std::unique_pointer for an array in
 * device-global memory.
 * 
 * @tparam T  an array type; _not_ the type of individual elements
 *
 * @param device on which to construct the array of elements
 * @param num_elements the number of elements to allocate
 * @return an ::std::unique_ptr pointing to the constructed T array
  * 
 */
```
And for every translation unit which includes this header, doxygen gives me:
```
/path/to/header.hpp:26: warning: explicit link request to 'std::unique_pointer' could not be resolved
```
I didn't make an explicit link request! 

Also strange is that I don't get the same warning about the second appearance of `std::unique_ptr`.

**To Reproduce**
Just make sure that comment above is stuck onto a a relevant function.

**Expected behavior**
No warning. Either there's auto-linkification (which there might be) or there isn't.

**Version**
1.11.0 (42d0f98dcbb309dcae08d1495786add326aaeb41*)

**Comment by albert-github** [comment]
Doxygen recognizes that `:: ...` is something that can be linked and does some automatic linking.

Based on the information provided I got with the `::std::unique_pointer` and the `::std::unique_ptr` a warning:
```
.../header.hpp:5: warning: explicit link request to 'std::unique_pointer' could not be resolved
.../header.hpp:12: warning: explicit link request to 'std::unique_ptr' could not be resolved
```

so:
- Can you please attach a, small, self contained example (source+configuration file in a, compressed, tar or zip file!) that allows us to reproduce the problem? Please don't add external links as they might not be persistent (also references to GitHub repositories are considered non persistent).

**Comment by eyalroz** [comment]
@albert-github : Attaching. Note that the doxygen.cfg is an auto-generated one with no changes.
[Uploading 10713.zip…]()

**Comment by eyalroz** [comment]
@albert-github : I don't know what the reminder frequency is on this issue tracker... so, ping about the example you requested.

**Comment by albert-github** [comment]
I don't see a neat solution for this problem only workaround I can see is to use `::%std::unique_ptr`
Maybe @doxygen has an idea

**Comment by doxygen** [comment]
@albert-github I think the warning is simply not correct in this case.

@eyalroz Please verify if the referenced commit fixes the problem for you. Do not close the issue, this will be done automatically when the next official release becomes available.

**Comment by eyalroz** [comment]
@doxygen Verified.

By the way, how do you automate the closing of issues by making releases?

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.11.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).
